package.json

```diff
@@ -141,7 +141,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint:fix && yarn format:staged"
+      "pre-commit": "yarn precommit"
     }
+  },
+  "lint-staged": {
+    "*.js": [
+      "yarn lint:fix",
+      "yarn prettier",
+      "git add"
```

Comment:
Does this automatically run `git add`?

Reply:
Yes, on the changed files (which are the staged files, anyway), so nothing breaks

Reply:
`changed` or `staged` files? I very much only use `git add -p` for adding only the changes I really want into the staging area. As I understand it, unstaged changes would be added to the staging area with this change?

Reply:
I apologize for the confusion. I actually meant that it runs on staged files, and after making fixes, it adds the files again, to be able to commit the new changes.

If ran without `git add` as a final task, the changes from the tasks would not be included in the commit.

Reply:
yeah nice :+1: